README.md

```diff
@@ -154,6 +154,7 @@ You can pass options either via command line (in [kebab-case](https://lodash.com
 
 These options are currently available:
 - `branch`: The branch on which releases should happen. Default: `'master'`
+- `no-pre-release`: The npm dist tags that shouldn't be pre-released on GitHub. Default: `['latest']`. Can also be set to a single string. Note that if `publishConfig.tag` value is **included within** this option **or set to** `latest` (it is by default) a **pre-release won't happen**
```

Comment:
I'm afraid that a **pre-release won't happen** might give the idea that the release will not be done.

Maybe something as follow instead ?

`no-pre-release`: By default, releases on an npm dist-tag other than `latest` will produce a [Github release](https://help.github.com/articles/creating-releases) with the pre-release tag. You can add any dist-tag in the `no-pre-release` array in order to produce a regular (non pre-release) on Github for those dist-tag.